src/app/base/components/node/NodeTestDetails/NodeTestDetails.test.tsx

```diff
@@ -47,10 +47,12 @@ describe("NodeTestDetails", () => {
       <Provider store={store}>
         <MemoryRouter initialEntries={["/machine/abc123/testing/1/details"]}>
           <CompatRouter>
-            <Route
-              path="/machine/:id/testing/:scriptResultId/details"
-              render={() => <NodeTestDetails getReturnPath={getReturnPath} />}
-            />
+            <Routes>
```

Comment:
A lot of components had to be updated to use the RRv6 versions of `useLocation` and `useParams` which meant a lot of tests had to be wrapped in Routes.